+ Created and deleted temporary directories with root permissions as tests were failing
+ to create directories using nobody.
diff --git a/testcases/kernel/syscalls/chmod/chmod04.c b/testcases/kernel/syscalls/chmod/chmod04.c
index cbc13cf61..b52845021 100644
--- a/testcases/kernel/syscalls/chmod/chmod04.c
+++ b/testcases/kernel/syscalls/chmod/chmod04.c
@@ -162,13 +162,14 @@ void setup(void)
        tst_sig(NOFORK, DEF_HANDLER, cleanup);

        tst_require_root();
+       tst_tmpdir(); // TODO: Will removed once git issue :357 is fixed
        ltpuser = getpwnam(nobody_uid);
        if (setuid(ltpuser->pw_uid) == -1)
                tst_resm(TINFO | TERRNO, "setuid(%u) failed", ltpuser->pw_uid);

        TEST_PAUSE;

-       tst_tmpdir();
+//     tst_tmpdir(); // TODO: Will be enabled once git issue:236 is fixed

        /*
         * Create a test directory under temporary directory with specified
@@ -185,7 +186,7 @@ void setup(void)
  */
 void cleanup(void)
 {
-
+       tst_require_root(); // TODO: Will removed once git issue :357 is fixed
        tst_rmdir();

 }
